bin/casher

```diff
@@ -88,10 +88,16 @@ class Casher
   end
 
   def tar(flag, file, *args, &block)
-    command = "tar -Pj#{flag}f #{Shellwords.escape(file)} #{Shellwords.join(args)}"
+    compression_flag = file.end_with?('.tbz') ? 'j' : 'z'
+    command = "tar -P#{compression_flag}#{flag}f #{Shellwords.escape(file)} #{Shellwords.join(args)}"
     block ||= proc { puts "FAILED: #{command}", File.read("#{@casher_dir}/tar.err.log"), File.read("#{@casher_dir}/tar.log") }
     block.call unless system "#{command} 2>#{@casher_dir}/tar.err.log >#{@casher_dir}/tar.log"
   end
+
+  private
+  def download_from(urls)
+    (urls.map {|url| url.gsub(/\.tbz(\?)?\b/, '.tgz\1')} + urls).uniq
```

Comment:
Actually, there is a chance that this is insufficient; changing the file's extension may change the required signature for fetching.

If this is indeed the case, URL's must be constructed on the `travis-build` side.